gym/utils/play.py

```diff
@@ -120,11 +129,11 @@ def display_arr(
 
 def play(
     env: Env,
+    keys_to_action: Dict[Union[Tuple[Union[str, int]], str], ActType],
```

Comment:
This seems like an unnecessary breaking change? (changing the order of arguments)

Reply:
I spoke to @Markus28 about it. This was a bug in the previous implementation as `keys_to_action` is a required parameter if you look at the for loop after the docstring.

Reply:
I think I would actually prefer to leave `key_to_action` optional and try to fetch it from the environment, then throw an exception if that's not possible. That's how it used to work (before I kinda unknowingly broke it with the for loop).

Reply:
Also, I need to update the tests for utils.play, in particular they currently don't seem to cover the case where `env` provides keys_to_action.

Reply:
Im going to remove `utils.play` from this PR and let @Markus28 fix it in another PR